pkgs/tools/inputmethods/input-remapper/default.nix

```diff
@@ -45,11 +45,6 @@ in
     hash = "sha256-KPQLgXSonuOgphagYN2JN+CMIpmjTIPUTCqOPDk0UYU=";
   };
 
-  # Fixes error
-  # Couldn’t recognize the image file format for file "*.svg"
```

Comment:
That sounds more like gdk-pixbuf issue.

Reply:
the issue was that `GDK_PIXBUF_MODULE_FILE` pointed to the gdk-pixbuf module file instead of the one in librsvg but that should be fixed since https://github.com/NixOS/nixpkgs/pull/208537

Reply:
before
```
    --set 'GDK_PIXBUF_MODULE_FILE' '/nix/store/mihzapwslgszs2amsvkywb7qnhnxb5zm-gdk-pixbuf-2.42.10/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache' \
```

after
```
    --set 'GDK_PIXBUF_MODULE_FILE' '/nix/store/1pkbzqhij5ay6yd7q1vj5qk4rxyi4rkc-librsvg-2.55.1/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache' \
```